charts/test/tigera_operator_chart_test.go

```diff
@@ -0,0 +1,112 @@
+// Package charttest uses 'helm template' to render the helm package with various input values,
+// unmarshals the resulting yaml into kubernetes resource types, and then tests that the correct fields
+// are set accordingly.
+package charttest
+
+import (
+	"path/filepath"
+
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/gruntwork-io/terratest/modules/helm"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Tigera Operator Helm Chart", func() {
```

Comment:
You've already written it, so I won't make you change it, but I think we've said all new test suites should be written in standard `go test` without ginkgo / gomega, in case you're interested in tweaking these.